java/trace/src/main/scala/org/typelevel/otel4s/java/trace/Traces.scala

```diff
@@ -28,11 +30,20 @@ trait Traces[F[_]] {
 
 object Traces {
 
-  def ioLocal[F[_]: LiftIO: Sync](jOtel: JOpenTelemetry): F[Traces[F]] =
-    for {
-      provider <- TracerProviderImpl.ioLocal(jOtel.getTracerProvider)
-    } yield new Traces[F] {
+  def local[F[_]](
+      jOtel: JOpenTelemetry
+  )(implicit F: Sync[F], L: Local[F, Scope]): Traces[F] = {
+    val provider = TracerProviderImpl.local(jOtel.getTracerProvider)
+    new Traces[F] {
       def tracerProvider: TracerProvider[F] = provider
     }
+  }
+
+  def ioLocal[F[_]: LiftIO: Sync](jOtel: JOpenTelemetry): F[Traces[F]] =
```

Comment:
This is the last thing in the public API above cats-effect-std.  It feels like it doesn't belong, but it adds boilerplate to every main to remove it.  Thoughts?

We can't consider removing the dependency until the `Local[IOLocal, E]` instance finds another upstream home, presumably in cats-effect.